modules/calc/src/main/java/com/opengamma/strata/calc/runner/function/result/CurrencyValuesArray.java

```diff
@@ -63,6 +63,17 @@
    * @return an instance with the specified currency and values
    */
   public static CurrencyValuesArray of(Currency currency, double[] values) {
+    return new CurrencyValuesArray(currency, DoubleArray.copyOf(values));
+  }
```

Comment:
I'd prefer to remove this factory method, which is what happened on other classes like curves. (Caller creates `DoubleArray`)